bundlr/manifest.go

```diff
@@ -0,0 +1,16 @@
+package bundlr
+
+const (
+	manifestFilename = "manifest.json"
+)
+
+// UserData allows to store any KV information
+type UserData map[string]interface{}
+
+// Manifest stores bundle metadata
+// It is written as a json file at the root level of the bundle
+type Manifest struct {
+	Version       string    `json:"version"`
+	ApplicationID string    `json:"applicationid"`
```

Comment:
We don't need this. When I mentioned "ApplicationID" I meant a particular application should use a unique "ApplicationID" as the UserData key.

Reply:
Okay so that KV can be used or not but ultimately it's at the user's discretion. I have removed it, thanks.

Reply:
We call it "UserData", but it's most likely be handled by the "UserAgent" which meant the application handling the Bundle.

The "UserData" will be look like: 

```json
{
    "userdata": {
        "com.example.foo": ...
    } 
}
```